zipkin-server/src/main/resources/zipkin-server-shared.yml

```diff
@@ -121,6 +121,13 @@ zipkin:
       db: ${MYSQL_DB:zipkin}
       max-active: ${MYSQL_MAX_CONNECTIONS:10}
       use-ssl: ${MYSQL_USE_SSL:false}
+    influxdb:
+      url: ${INFLUXDB_URL:http://localhost:8086}
+      database: ${INFLUXDB_DB:zipkin}
+      measurement: ${INFLUXDB_MEASUREMENT:zipkin}
+      retentionPolicy: ${INFLUXDB_RETETION_POLICY:default}
+      username: ${INFLUXDB_USERNAME:root}
+      password: ${INFLUXDB_PASSWORD}
```

Comment:
I think you want INFLUXDB_PASSWORD: (trailing colon which makes it default to empty string)